terraform/modules/secret_manager/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "project_id" {
+  description = "GCP project ID where the secrets belongs."
+  type        = string
+}
+
+variable "secret_id" {
+  description = <<EOT
+    ID of the secret. This must be unique within the project. This is the secret
+    name.
+  EOT
+  type        = string
+}
+
+variable "labels" {
+  description = "The labels assigned to this Secret."
+  type        = map(string)
+  default     = {}
+}
+
+variable "jira_api_token_accessors" {
```

Comment:
This only allows reading the secret, right? You would need somebody to be an admin https://cloud.google.com/secret-manager/docs/access-control#secretmanager.admin to add/update JIRA cred in the secret, no?

Reply:
I was hesitating whether I need to provide ability to add each type of role or just add minimum things for the server to be functioning. Added ability to set admin role for now.

Reply:
Note that secretAccessor and secretReader are different IAM permissions. Reader can see metadata, but not the secret. Accessor can access the raw secret.

Reply:
Thanks @sethvargo for the details :D